presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry.py

```diff
@@ -170,23 +170,22 @@ def get_recognizers(
                 if language == rec.supported_language
             ]
         else:
-            for entity in entities:
-                subset = [
-                    rec
-                    for rec in all_possible_recognizers
-                    if entity in rec.supported_entities
-                    and language == rec.supported_language
-                ]
-
-                if not subset:
-                    logger.warning(
-                        "Entity %s doesn't have the corresponding"
-                        " recognizer in language : %s",
-                        entity,
-                        language,
-                    )
-                else:
-                    to_return.update(set(subset))
+            entities = set(entities)
```

Comment:
`entity` is later used. How about:
```suggestion
            entities = set(entities)
            for entity in entities:
```